raiden/settings.py

```diff
@@ -63,7 +63,7 @@
 PRODUCTION_CONTRACT_VERSION = raiden_contracts.constants.CONTRACTS_VERSION
 DEVELOPMENT_CONTRACT_VERSION = raiden_contracts.constants.CONTRACTS_VERSION
 
-MIN_REI_THRESHOLD = 100
+MIN_REI_THRESHOLD = TokenAmount(55 * 10 ** 17)  # about 1.1$
```

Comment:
So maybe it starts making sense to have a `USD_TO_REI` ratio and use that. These values are hard to read.